### ISSUE TITLE
New Developer Update 10.15.3 beta 3	appears to delete SideCar and the work-around no longer works

### ISSUE THREAD

**Issue by timothystutters** [ben-z/free-sidecar#37]
The subject says it all. I should have thought about this before updating, but there it is. I lost sidecar, reinstalled it per the instructions, but it does not seem to take anymore.

**Comment by ben-z** [comment]
Can you share your `SidecarCore` file?

**Comment by timothystutters** [comment]
[SidecarCore.zip](https://github.com/ben-z/free-sidecar/files/4115723/SidecarCore.zip)
The file is attached as a Zip here.

**Comment by pookjw** [comment]
Can you see Sidecar on System Preferences app after patch?

**Comment by timothystutters** [comment]
No, Sidecar completely disappeared from the System Preferences after the patch!

**Comment by ben-z** [comment]
@timothystutters that file appears unpatched. Can you share the patched file?

**Comment by timothystutters** [comment]
[SidecarCore.zip](https://github.com/ben-z/free-sidecar/files/4118845/SidecarCore.zip)
Okay -- this should be the patched file. That first file was from my renewed attempt yesterday to re-download SidecarCore and restore it to my machine. The zip file attached here contains the file I used about a month ago to install Sidecar on my machine.

**Comment by ben-z** [comment]
@timothystutters That still looks like the unpatched file to me 😅 . You can verify that it's patched by opening it with free-sidecar and see if any of the buttons are green.

**Comment by timothystutters** [comment]
Well this is strange... I looked at the old file, and as you said, none of the buttons were green. So I clicked on my Macbook Pro with the old file (11,2), and it is now working again. I now have Sidecar, and it seems to work as well as it did before. I really have no explanation for this, unless the patch just reset the old SideCarCore file. As for why the newer SideCarCore file didn't work, I really have no theories for that! Thanks for your help!
